docs/CppInteropability/GettingStartedWithC++Interop.md

```diff
@@ -0,0 +1,217 @@
+#  Getting started with C++ Interoperability
+
+This document is desgined to get you started with bidirectional API-level interoperability between Swift and C++.
+
+## Table of Contents
+
+- [Creating a Module to contain your C++ source code](#creating-a-module-to-contain-your-c-source-code)
+- [Adding C++ to an Xcode project](#adding-c-to-an-xcode-project)
+- [Creating a Swift Package](#Creating-a-Swift-Package)
+- [Building with CMake](#building-with-cmake)
+
+## Creating a Module to contain your C++ source code
+
+- Create a new C++ implementation and header file
+- For this example we will call the files Cxx, so we should have a Cxx.cpp and Cxx.hpp.
+- Create a module.modulemap file to expose your source code 
+- In order to expose our C++ code we must designate a separate as the source.
+- Create an empty file and call it `module.modulemap`
```

Comment:
This and line 16 seem redundant. Maybe they could be joined. I'm not sure you need line 17.